integration_tests/test_data.py

```diff
@@ -47,6 +47,10 @@
 FIJI_PATH = os.environ.get(
     'FIJI_PATH', '/allen/aibs/pipeline/image_processing/volume_assembly/Fiji.app')
 
+ROUGH_SOLVER_EXECUTABLE = os.environ.get(
```

Comment:
I think we should point this to the auto-deployed version of the dev solver, which is at /allen/aibs/pipeline/image_processing/volume_assembly/EMAligner/dev/allen_templates/do_rough_alignment, as then the integration test will validate that the most up to date compiled and dev deployed version of the EM_aligner codebase 'works'.

Reply:
commenting this out as it is irrelevant for this module. Will make changes appropriately in rough_align_integration_test branch.